extensions/graphql/src/__tests__/fixtures/graphql-test/src/services/recipe.service.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/graphql
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {asService, bind, inject} from '@loopback/core';
+import {plainToClass} from 'class-transformer';
+import {RecipeInput} from '../graphql-types/recipe-input';
+import {Recipe} from '../graphql-types/recipe-type';
+
+@bind(asService(RecipeService))
+export class RecipeService {
+  private autoIncrementValue: number;
+
+  constructor(@inject('recipes') private readonly items: Recipe[]) {
+    this.autoIncrementValue = this.items.length;
+  }
+
+  async getAll() {
+    return this.items;
+  }
+
+  async getOne(id: string) {
+    return this.items.find(it => it.id === id);
```

Comment:
`RecipeService` looks like a poor-man's version of `RecipeRepository`. I am confused why you are writing this new service and not using `RecipeRepository` instead?

Reply:
Yes.